src/components/LoginPage.tsx

```
@@ -26,62 +30,86 @@ export function LoginPage({ onLogin }: LoginPageProps) {
 
-          <div className="text-sm text-base-content/70">
-            <ol className="list-decimal list-inside space-y-2">
-              <li>Acesse GitHub Settings → Developer settings</li>
-              <li>Personal access tokens → Tokens (classic)</li>
-              <li>Generate new token (classic)</li>
-              <li>Selecione as permissões: <code className="bg-base-200 px-1 rounded">repo</code></li>
-              <li>Copie e cole o token aqui</li>
-            </ol>
+            {onSkip && (
+              <div className="alert alert-warning mt-4">
+                <svg xmlns="http://www.w3.org/2000/svg" className="stroke-current shrink-0 h-6 w-6" fill="none" viewBox="0 0 24 24"><path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M12 9v2m0 4h.01m-6.938 4h13.856c1.54 0 2.502-1.667 1.732-3L13.732 4c-.77-1.333-2.694-1.333-3.464 0L3.34 16c-.77 1.333.192 3 1.732 3z" /></svg>
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)